packages/cms/pages/settings/team-members/team-members.tsx

```diff
@@ -117,24 +135,60 @@ export const TeamMembers: FunctionComponent<ProfileProps> = () => {
         actions: [
           {
             name: 'Edit',
-            description: 'Edit this item',
+            description: 'Change user role',
             icon: 'pencil',
             type: 'icon',
             onClick: item => {}
           },
           {
             name: 'Delete',
-            description: 'Delete this item',
+            description: 'Remove this user',
             icon: 'trash',
             type: 'icon',
             color: 'danger',
-            onClick: item => {}
+            disabled: true,
```

Comment:
I think this should be disabled conditionally ?

Reply:
It wasn't even needed, it didn't work